### ISSUE TITLE
Add code coverage tag into README.md

### ISSUE THREAD

**Issue by dariusstrasel** [mechamecho/DataDrivenTesting#19]

**Comment by mechamecho** [comment]
Would we be using something like [this](https://github.com/OpenCover/opencover) for the code coverage tag ?

**Comment by dariusstrasel** [comment]
Yes, OpenCover is a good example. Ideally we'd want to hook this into our CI server (Jenkins) and have it calculate our coverage anytime we have a merge/PR/new release.
SonarCube is probably a better one to look into.
